### PR TITLE
Decommission old /whatsnew and /firstrun pages for Dev Edition (Fixes #7437)

### LINKED ISSUE #7437: Decommission old /whatsnew and /firstrun pages for Dev Edition.
## Description

These pre-quantum pages are now pretty old, and are still built using Sandstone & Less. I think they likely pretty safe to remove, and the URLs can default to the generic /whatsnew and /firstrun templates that get shown for release versions.

- https://www.mozilla.org/en-US/firefox/56.0a2/whatsnew/
- https://www.mozilla.org/en-US/firefox/56.0a2/firstrun/

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] URLs default to the generic /whatsnew and /firstrun templates in production.

#### Tasks
- [ ] Remove templates and assets.
- [ ] Update tests where needed.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description

Removes old whatsnew and firstrun pages for Developer Edition.

- http://localhost:8000/en-US/firefox/56.0a2/whatsnew/
- http://localhost:8000/en-US/firefox/56.0a2/firstrun/

## Issue / Bugzilla link
#7437

## Testing
- [x] URLs should show default whatsnew and firstrun templates instead of custom pages.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
There are a few files in media/img/developer which these pages were the only user of:

- feature-responsive.jpg
- feature-valence.jpg
- title-inverse.png
- title-inverse-high-res.png

File: bedrock/firefox/tests/test_base.py [hunk 1]
```diff
@@ -286,11 +286,11 @@ def test_fx_beta_68_0_trailhead_non_locales(self, render_mock):
 
     @override_settings(DEV=True)
     def test_fx_dev_browser_35_0_a2_whatsnew(self, render_mock):
-        """Should show dev browser whatsnew template"""
+        """Should defaultr whatsnew template"""
```
Comment (stephaniehobson) [inline bedrock/firefox/tests/test_base.py]: typo.

**Comment by alexgibson** [comment]
Thanks @stephaniehobson updated!

File: bedrock/firefox/tests/test_base.py [hunk 2]
```diff
@@ -286,7 +286,7 @@ def test_fx_beta_68_0_trailhead_non_locales(self, render_mock):
 
     @override_settings(DEV=True)
     def test_fx_dev_browser_35_0_a2_whatsnew(self, render_mock):
-        """Should defaultr whatsnew template"""
+        """Should default whatsnew template"""
```
Comment (stephaniehobson) [inline bedrock/firefox/tests/test_base.py]: I'm not sure what this comment is supposed to communicate. Can you update it for clarity please?

Comment (alexgibson) [reply bedrock/firefox/tests/test_base.py]: Doh sorry, will update!

**Comment by alexgibson** [comment]
@stephaniehobson updated, let's try again :)

**Pull request merged**